examples/demo-rn-expo/.env.example

```
@@ -4,22 +4,22 @@ EXPO_PUBLIC_REOWN_APP_ID=
 
-# (Required) The target URI required by the protocol for commenting
+# (Optional) The target URI required by the protocol for commenting
 EXPO_PUBLIC_TARGET_URI=http://localhost:3000
 
 # (Required) The RPC URL for the selected chain
 EXPO_PUBLIC_RPC_URL=https://base-mainnet.g.alchemy.com/v2/Abcdefg
 
-# (OPTIONAL) If omitted the comment author will not render as link
+# (Optional) If omitted the comment author will not render as link
 # use {address} for commenter's address
 EXPO_PUBLIC_COMMENT_AUTHOR_URL="https://app.interface.social/{address}"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)